### DISCUSSION TITLE
bookmarklet?

### DISCUSSION THREAD

**Discussion by sttrebo** [TomBursch/kitchenowl#303] [Q&A]
not sure if this is the right place or not, but i was curious if there was a browser extension or bookmarklet to scrape a url for the recipe import.  or do i have to go thru the import from url dialog?

**Comment by TomBursch** [comment]
I've never heard of bookmarklets before, looks interesting. There sadly isn't such thing. On a mobile device, you can share the link to KitchenOwl which will open the scrape page. Feel free to open a feature request issue for this.
